### ISSUE TITLE
Protocol-less image URLs aren't handled properly

### ISSUE THREAD

**Issue by micyoung70** [chuckreynolds/WPFBOGP#68]
We use <img> tags with src="//imghostname/path_to/xyz.jpg" URLs that don't have the protocol. This allows HTTP pages to use an HTTP URL and HTTPS pages to use an HTTPS URL.

The WPFBOG plugin turns this into:
```
<meta property="og:image" content="https://bloghostname/blogpath/imghostname/path_to/xyz.jpg"/>
```

The preg_match() call on line 57 needs to be modified to check for a valid "//" prefix URL match.

**Comment by chuckreynolds** [comment]
I agree but when I originally made this plugin, Facebook and the few sites that "supported" open graph didn't validate such urls. I know reddit does now as of last year based on a patch in /r/bugs and maybe FB does now too but that's something I'd have to test. But good heads up - it's at least time to retest that and see how every major site works with that.  
Sites I typically test:  Facebook, Twitter, Linkedin, Pinterest, Reddit.
